src/global_styling/variables/_colors.ts

```diff
@@ -31,25 +31,23 @@ export const poles = {
 };
 
 const textVariants = {
-  textPrimary: computed(['colors.primary'], ([primary]) =>
-    makeHighContrastColor(primary)
-  ),
-  textAccent: computed(['colors.accent'], ([accent]) =>
-    makeHighContrastColor(accent)
-  ),
-  textSuccess: computed(['colors.success'], ([success]) =>
-    makeHighContrastColor(success)
-  ),
-  textWarning: computed(['colors.warning'], ([warning]) =>
-    makeHighContrastColor(warning)
-  ),
-  textDanger: computed(['colors.danger'], ([danger]) =>
-    makeHighContrastColor(danger)
-  ),
-  textDisabled: computed(['colors.disabled'], ([disabled]) =>
-    makeDisabledContrastColor(disabled)
-  ),
-  link: computed(['colors.textPrimary'], ([textPrimary]) => textPrimary),
+  textPrimary: computed((theme) => makeHighContrastColor(theme.colors.primary)),
+  textAccent: computed(([accent]) => makeHighContrastColor(accent), [
+    'colors.accent',
+  ]),
```

Comment:
This might have answered my question above. Just want to know what the preferred guidance is for these computed values. Is one method better than the other?

Reply:
It's entirely up to consumer preference. I'm not sure there's a large enough computational difference to say that one is better than the other.